app/assets/javascripts/admin/addon/components/site-settings/file-types-list.js

```diff
@@ -0,0 +1,76 @@
+import Component from "@ember/component";
```

Comment:
Maybe you couldn't do this because of how this site setting stuff is structured, but if possible this new component should be a `.gjs` file and import from `@glimmer/component` :)

Reply:
Plus side to doing this is that you will no longer need the `set/get` calls and you can use `@tracked` properties